src/main/java/org/jabref/logic/citationstyle/JabRefItemDataProvider.java

```diff
@@ -191,7 +191,7 @@ public String toJson() {
         return entries.stream()
                 .map(entry -> bibEntryToCSLItemData(entry, bibDatabaseContext, entryTypesManager))
                 .map(item -> item.toJson(stringJsonBuilderFactory.createJsonBuilder()))
-                .map(item -> (String) item)
+                .map(String.class::cast)
```

Comment:
Maybe this should have a comment that CSLItemData internally always provides a String?
Or should we test it before downcasting?